packages/web/src/components/pages/Webhooks/WebhookDetails/ActionTab/CustomPayload.tsx

```diff
@@ -44,6 +46,14 @@ const CustomPayload: FC = () => {
       initialValues={{payloadTemplate: current!.payloadTemplate || ''}}
       disabled={!mayEdit}
       onConfirm={onFinish}
+      footer={
+        <>
+          Learn more about{' '}
+          <ExternalLink href={externalLinks.notificationsAndWebhooks} target="_blank">
```

Comment:
Could you point to the custom payload section?

https://docs.testkube.io/articles/webhooks/#webhook-payload